src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,29 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo,
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    id, completed, title, user,
+  } = todo;
+
+  return (
+    <article
+      data-id={id}
+      className={classNames('TodoInfo',
+        { 'TodoInfo--completed': completed })}
+    >
+      <h2 className="TodoInfo__title">
+        {title}
+      </h2>
+
+      {user
+        ? <UserInfo user={user} />
+        : 'Userus Anonymus'}
```

Comment:
Good idea. But we will never see the message 'Userus Anonymus', we will see the error message